### PR TITLE
fixed broken link for 13408

### LINKED ISSUE #13408: Broken link in documentation
### Description

The last paragraph of  react-native/docs/Networking.md contains a link to a page recently removed in 08dbc43fa64ff1dfc0d364b6cef6262426f76b64

### Reproduction Steps and Sample Code

Click on 'Navigators' in the last paragraph of https://facebook.github.io/react-native/docs/network.html

### Solution

I didn't PR it since I don't know how you'd want to rephrase the conclusion.

### Additional Information

* React Native version: 0.43

### PR DESCRIPTION
**Pull request by jordanpapaleo**
Thanks for submitting a PR! Please read these instructions carefully:

- [ ] Explain the **motivation** for making this change.
- [ ] Provide a **test plan** demonstrating that the code is solid.
- [ ] Match the **code formatting** of the rest of the codebase.
- [ ] Target the `master` branch, NOT a "stable" branch.

## Motivation (required)

Fixes #13408

There was a broken link in the docs

## Test Plan (required)

- Navigate to https://facebook.github.io/react-native/docs/network.html
- Go to the last paragraph and click on the navigators link
- It will bring you to https://facebook.github.io/react-native/docs/navigation.html

### CODE REVIEW

File: docs/Navigation.md [hunk 1]
```diff
@@ -134,4 +134,4 @@ class MyScene extends React.Component {
 }
 ```
 
-Check out the [`NavigatorIOS` reference docs](docs/navigatorios.html) to learn more about this component.
+Check out the [`NavigatorIOS` reference docs](docs/navigation.html) to learn more about this component.
```
Comment (shergin) [inline docs/Navigation.md]: This change seems completely unrelated to test plan.

Comment (jordanpapaleo) [inline docs/Navigation.md]: Navigation iOS page no longer exists. I updated the docs to use the correct page. Not sure what you are asking.

**Comment by jordanpapaleo** [comment]
You have to run the website locally. 

Sent from my smallest computer

On May 28, 2017, at 11:53 AM, Valentin Shergin <notifications@github.com> wrote:

@shergin requested changes on this pull request.

The link mentioned in test plan is broken, but this PR does not fix that.

In docs/Navigation.md:

> @@ -134,4 +134,4 @@ class MyScene extends React.Component {
 }
 ```
 
-Check out the [`NavigatorIOS` reference docs](docs/navigatorios.html) to learn more about this component.
+Check out the [`NavigatorIOS` reference docs](docs/navigation.html) to learn more about this component.
This change seems completely unrelated to test plan.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by shergin** [comment]
@jordanpapaleo Your test plan mentions "network.html" page but your diff changes "Navigation.md"... 🤔

**Comment by jordanpapaleo** [comment]
The markdown files are used to generate the HTML. 

Sent from my smallest computer

On May 28, 2017, at 2:00 PM, Valentin Shergin <notifications@github.com> wrote:

@jordanpapaleo Your test plan mentions "network.html" page but your diff changes "Navigation.md"... 🤔

**Comment by jordanpapaleo** [comment]
Im off the grid for the moment.  Ill take a look first thing tomorrow :)

Thanks for looking at this.
